app/src/main/java/io/spaceapi/community/myhackerspace/Main.java

```diff
@@ -908,6 +1001,16 @@ private void populateDataHs() {
                 }
             }
 
+            if (data.linked_spaces.length > 0) {
+                addTitle(iftr, vg, R.string.api_linked_spaces);
+                for (LinkedSpace space : data.linked_spaces) {
+                    if (space.website != null) {
+                        TextView tv = addEntry(iftr, vg, space.website);
+                        Linkify.addLinks(tv, Pattern.compile("^.*$", Pattern.DOTALL), null);
+                    }
+                }
+            }
```

Comment:
Idea for the future: When tapping on one of the links, open that space directly in the app. (We can load spaces by URL, so by adding an intent for this, we should be able to open the endpoints directly.)

![image](https://github.com/user-attachments/assets/d74d3509-03ae-45e6-ba6c-654e64397e80)

(However, that can be moved into a separate GitHub issue, since it's a bit out of scope for this PR.)

Reply:
I tried this, but it did not work as expected. I think this will require a bigger change to the lifecycle of the Main activity. I didn't dig very deep so far though.